src/Symfony/Component/Notifier/Bridge/MessageMedia/composer.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "symfony/message-media-notifier",
+  "type": "symfony-bridge",
+  "description": "Symfony MessageMedia Notifier Bridge",
+  "keywords": ["sms", "messagemedia", "notifier"],
+  "homepage": "https://symfony.com",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Adrian Nguyen",
+      "email": "vuphuong87@gmail.com"
+    },
+    {
+      "name": "Symfony Community",
+      "homepage": "https://symfony.com/contributors"
+    }
+  ],
+  "require": {
+    "php": ">=7.2.5",
+    "symfony/http-client": "^4.3|^5.0|^6.0",
```

```suggestion
    "symfony/http-client": "^4.4|^5.2|^6.0",
```

I think we can go with these constraints